CONTRIBUTORS.rst

```diff
@@ -291,6 +291,7 @@ Contributors
 * Alejandro Garza
 * Rajeev J Sebastian
 * Coen van der Kamp
+* Arthur Holzner
```

Comment:
You're already on the list, four names above :-)

Reply:
eh.. yea
i ll fix this with the upcoming test